reporter/src/funTest/assets/static-html-reporter-test-input.yml

```diff
@@ -39,23 +39,43 @@ repository:
       - given: "GPL-2.0-only OR MIT"
         choice: "MIT"
       package_license_choices:
-        - package_id: "Maven:com.h2database:h2:1.4.200"
-          license_choices:
-            - given: "MPL-2.0 OR EPL-1.0"
-              choice: "MPL-2.0"
+      - package_id: "Maven:com.h2database:h2:1.4.200"
+        license_choices:
+        - given: "MPL-2.0 OR EPL-1.0"
+          choice: "MPL-2.0"
```

Comment:
Nit: This seems to be a pure formatting change unrelated to the model changes, but I'm ok with keeping it as part of this commit.

Reply:
That's correct, but I didn't want to spend time finding out which part of the diff is because of manual changes and which is because of model changes (the different sorting is probably also caused by manually added content), so I just regenerated the whole file all at once.